config/routes.rb

```diff
@@ -802,30 +791,44 @@
     post '/tasks/create-fake-provider' => 'tasks#create_fake_provider'
     post '/tasks/:task' => 'tasks#run', as: :run_task
     get '/tasks/confirm-delete-test-applications' => 'tasks#confirm_delete_test_applications', as: :confirm_delete_test_applications
     get '/tasks/confirm-cancel-applications-at-end-of-cycle' => 'tasks#confirm_cancel_applications_at_end_of_cycle', as: :confirm_cancel_applications_at_end_of_cycle
     get '/tasks/confirm-reject-applications-awaiting-references-at-end-of-cycle' => 'tasks#confirm_reject_applications_awaiting_references_at_end_of_cycle', as: :confirm_reject_applications_awaiting_references_at_end_of_cycle
 
-    get '/validation-errors' => 'validation_errors#index', as: :validation_errors
-    get '/validation-errors/search' => 'validation_errors#search', as: :validation_error_search
-    get '/validation-errors/summary' => 'validation_errors#summary', as: :validation_error_summary
+    scope '/performance' do
+      get '/' => 'performance#index', as: :performance
+      get '/course-stats', to: 'performance#course_stats', as: :course_stats
+      get '/course-options', to: 'performance#course_options', as: :course_options
+      get '/unavailable-choices' => 'performance#unavailable_choices', as: :unavailable_choices
+
+      get '/application-timings', to: 'performance#application_timings', as: :application_timings
+      get '/submitted-application-choices', to: 'performance#submitted_application_choices', as: :submitted_application_choices
+      get '/candidate-journey-tracking', to: 'performance#candidate_journey_tracking', as: :candidate_journey_tracking
+      get '/providers', to: 'performance#providers_export', as: :providers_export
+      get '/referee-survey', to: 'performance#referee_survey', as: :referee_survey
+      get '/candidate-survey', to: 'performance#candidate_survey', as: :candidate_survey
+      get '/active-provider-users', to: 'performance#active_provider_users', as: :active_provider_users
+      get '/course-choice-withdrawal', to: 'performance#course_choice_withdrawal', as: :course_choice_withdrawal_survey
+      get '/tad-provider-performance', to: 'performance#tad_provider_performance', as: :tad_provider_performance
+      get '/offer-conditions', to: 'performance#offer_conditions', as: :offer_conditions
+      get '/reference-types', to: 'performance#application_references', as: :application_references
+
+      get '/validation-errors' => 'validation_errors#index', as: :validation_errors
+      get '/validation-errors/search' => 'validation_errors#search', as: :validation_error_search
+      get '/validation-errors/summary' => 'validation_errors#summary', as: :validation_error_summary
+
+      resources :data_exports, path: '/data-exports' do
+        member do
+          get :download
+        end
+      end
+    end
+
+    get '/email-log', to: 'email_log#index', as: :email_log
+    get '/vendor-api-requests', to: 'vendor_api_requests#index', as: :vendor_api_requests
 
```

Comment:
The URLs that have changed will need a redirect like this:

```rb
    get '/from', to: '/to'
```

Reply:
These are the effected URLs:

* `/support/applications/unavailable-choices` → `/support/performance/unavailable-choices`
* `/support/validation-errors/*` → `/support/performance/validation-errors/*`
* `/support/data-exports/* → `/support/performance/data-exports/*`

Given only linked to within support, do we need redirects for these?

Reply:
In this particular case I think it's fine to not create redirects for the old URLs :+1: